tests/data/instances/mc_tuples_with_background_instance_test.py

```diff
@@ -0,0 +1,284 @@
+# pylint: disable=no-self-use,invalid-name
+
+from unittest import TestCase
+import numpy as np
+
+from deep_qa.data.data_indexer import DataIndexer
+from deep_qa.data.dataset import TextDataset
+from deep_qa.data.instances.mc_tuples_with_background_instance import MCTuplesWithBackgroundInstance, \
+    IndexedMCTuplesWithBackgroundInstance
```

Comment:
you can just repeat the import from the line above, like `from deep_qa.data.instances.mc_tuples_with_background_instance import IndexedMCTuplesWithBackgroundInstance`. In general i feel like the `\` is inelegant, but that might just be me.